packages/notification-center/src/hooks/use-unseen-count-controller.hook.ts

```diff
@@ -7,18 +7,26 @@ export function useUnseenController() {
   const { api } = useApi();
   const { token } = useAuth();
 
+  const setUnseenCountCallback = useCallback(
+    (count: number) => {
+      setUnseenCount(count);
+      document.dispatchEvent(new CustomEvent('novu:unseen_count_changed', { detail: count }));
+    },
+    [setUnseenCount]
+  );
```

Comment:
when the `setUnseenCount` callback is called it will set the state and dispatch the custom event that we are listening to in the Vue component